### PR TITLE
Feature: basic UI

### PR DESCRIPTION
**Pull request by nachosala89**
- Added the main menu.
- Added method to list books, people, and rentals.
- Added method to create books, people, and rentals.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    permission = (permission.downcase == 'y')
+    person = Student.new(age, name, permission)
+  when '2'
+    puts 'Specialization:'
+    specialization = gets.chomp
+    person = Teacher.new(age, specialization, name)
+  end
+  people.push(person)
+  puts 'Person created successfully!' if people.include?(person)
+end
```
Comment (D4nielJ) [inline main.rb]: - [x] Creating a person object is failing. I think it's better if we try to fix this. I recommend you use only keyword arguments if that helps `keyword:`. An example of creating a student would be `Student.new(age: age, name: name, permission: permission)`. 

If you want to keep the classic no-named arguments, make sure that if you any keyword argument on the class definitions that you pass it correctly when you instantiate the class.

school-failing-student

Comment (D4nielJ) [reply main.rb]: Note to the next reviewer: Make sure to test for creating rentals and listing them, as I can't crate rentals without creating persons.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,120 @@
+require './person'
+require './student'
+require './teacher'
+require './book'
+require './rental'
```
Comment (D4nielJ) [inline main.rb]: [OPTIONAL] I think It's a good practice for the Ruby programmers to import their files with `require_relative` when they are not imported from gems or installed packages. [You can read more about good practices for Ruby here.](https://rubystyle.guide/)

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @nachosala89, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: :rocket: 
- The app initiates correctly!
- The menu works and displays the correct options

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @nachosala89 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**